src/index/io_flags.rs

```diff
@@ -0,0 +1,24 @@
+//! Module containing the io flags.
+
+/// Io Flags used during index reading, not all flags applicable to all indices
+// This is a set of constants rather than enum so that bitwise operations exist
+pub mod io_flag {
```

Comment:
Referred to [c_api here](https://github.com/facebookresearch/faiss/blob/main/c_api/index_io_c.h#L34).